hack/lib/common.sh

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env bash

```

Comment:
How is this common.sh? It has a lot of code similar to arktos-up but I don't see any change in arktos-up. What is the relationship? How is the script used? If you plan to refactor arktos-up, please do it in same commit. Also please include detailed user manual.

Reply:
I removed some common functions in arktos-up.sh
The script is for reusing the existing function in arktos-up.sh.